coffit/__init__.py

```diff
@@ -1,8 +1,7 @@
+"""Init."""
+
 import socketio
 
 sio = socketio.Server()
 
-from . import models  # noqa
-from . import api, auth  # noqa
```

Comment:
Without this, the SocketIO "endpoints" (`/auth` and `/`) don't get registered. It's either this or some sort of fancy `register_namespace` function...